sklearn/linear_model/ridge.py

```diff
@@ -244,8 +244,8 @@ def ridge_regression(X, y, alpha, sample_weight=None, solver='auto',
           (possibility to set `tol` and `max_iter`).
 
         - 'lsqr' uses the dedicated regularized least-squares routine
-          scipy.sparse.linalg.lsqr. It is the fastest but may not be available
-          in old scipy versions. It also uses an iterative procedure.
+          scipy.sparse.linalg.lsqr. It is the fastest and uses an iterative
```

Comment:
Not sure what to write here, suggestions more than welcome! I guess "it is the fastest" without more details is not something I would trust and I am certainly no expert on this.